### PR TITLE
Design styling forms

### PR DESCRIPTION
**Pull request by sulaimancode**
Added custom forms and styled them. Forms can take in additional fields(name) without having to change any css.

### CODE REVIEW

**Comment by motri** [comment]
Text is not as readable as it could be, maybe a change of background color?

**Review by motri — CHANGES_REQUESTED** [review]
The text is not as readable as it could, lighter font or background colour would help.

**Review by motri — APPROVED** [review]

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -13,3 +13,101 @@
...
  */
+#new_user {
+  margin: 50px auto;
+  width: 280px;
+  padding: 33px 25px 29px;
+  background: #AA1925;
+  border-bottom: 1px solid #c4c4c4;
+  border-radius: 5px;
+  box-shadow: 0 1px 5px rgba(0, 0, 0, 0.25);
+}
```
Comment (neoeno) [inline app/assets/stylesheets/application.css]: In general when styling it's best to add a class to the element and then use that to target styling — classes for styling, IDs (and sometimes other things) for Javascript targetting.

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -13,3 +13,101 @@
...
+  font-size: 17px;
+  border: 2px solid #ebebeb;
+  border-radius: 4px;
+  box-shadow: inset 0 -2px #ebebeb;
+}
+
+#user_email {
+  color:#666;
+  content:"You better do something";
+}
```
Comment (neoeno) [inline app/assets/stylesheets/application.css]: I think this `content` directive doesn't do anything, right?

File: app/assets/stylesheets/application.css [hunk 3]
```diff
@@ -13,3 +13,101 @@
...
+
+#sign-up-button {
+  background-color: #AA1925;
+}
+
+
+
```
Comment (neoeno) [inline app/assets/stylesheets/application.css]: A few blank lines here

File: app/views/users/new.html.erb [hunk 4]
```diff
@@ -1,28 +1,25 @@
...
-  <h2><%= t('.title') %></h2>
+  <h2>Sign up Now!!</h2>
 
   <%= form_for @user do |form| %>
-    <div class='text-field'>
-      <%= form.label :name %>
-      <%= form.text_field :name, :type => 'name' %>
+
+    <div class='text-field info'>
+      <%= form.text_field :name, :type => 'name', placeholder: 'Name..' %>
```
Comment (neoeno) [inline app/views/users/new.html.erb]: I'm a bit of a stickler for detail — but the rest of the fields have `...` and this one has `..` — probably best to stick to 3! There are a few instances of having just 2 throughout I believe?

File: app/assets/stylesheets/application.css [hunk 5]
```diff
@@ -13,3 +13,101 @@
  *= require_tree .
  *= require_self
  */
+#new_user {
```
Comment (neoeno) [inline app/assets/stylesheets/application.css]: I'd break this out to another file. `application.css` can grow out of control and become terrible for merge conflicts if you're not careful.

File: app/views/sessions/new.html.erb [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+  <div class="submit-field">
+    <%= form.submit %>
+  </div>
+
+  <div class="other-links">
+    <%= link_to ("Sign up"), sign_up_path %>
+    <%= link_to ("Forgot password?"), new_password_path, id: 'forgot' %>
+  </div>
+<% end %>
```
Comment (neoeno) [inline app/views/sessions/new.html.erb]: This needs indenting another step.

**Review by neoeno — CHANGES_REQUESTED** [review]
Nice work. A few comments re: CSS conventions & file organisation.

**Comment by sulaimancode** [comment]
Made all the required changes. Added an additional scss file for forms styling, used classes to target styling rather than ids, removed unnecessary lines of code and fixed some indentation.

**Review by neoeno — APPROVED** [review]

**Pull request merged**